pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -113,32 +112,20 @@ def find_infectees(self, infector_cell: Cell,
         """
         infectee_list = []
         # Chooses a list of cells (with replacement) for each infection
-        # event to occur in. Specifically inter-cell infections
-        # so can't be the same cell.
+        # event to occur in.
+        # Specifically inter-cell infections so can't be the same cell.
         distance_weights = []
-        for cell2 in possible_infectee_cells:
-            try:
-                distance_weights.append(1/DistanceFunctions.dist(
-                            infector_cell.location, cell2.location))
-            except ZeroDivisionError:
-                # If cells are on top of each other use nan placeholder
-                distance_weights.append(np.nan)
-        # Cells on top of each currently have a distance weight equal
-        # to the maximum of all other weights.
-        # Possibly want to do twice this.
-        number_of_nans = sum(np.isnan(distance_weights))
-        if number_of_nans == len(distance_weights):
-            distance_weights = [1 for _ in distance_weights]
-        elif number_of_nans > 0:
-            max_weight = np.nanmax(distance_weights)
-            distance_weights = np.nan_to_num(distance_weights,
-                                             nan=max_weight)
         # Use of the cutoff distance idea from CovidSim.
         cutoff = Parameters.instance().infection_radius
```

Comment:
I think you don't need 'cutoff' here, as you defined it in the near_neighbour function

Reply:
I think then we change the exception - the easiest thing here is to generate it on the fly in the exception rather than storing it in here. However you may want to reword the wording of the logging message given the underlying code has changed, to reflect the fact that the cell has no neighbours (and then you could add a logging message in the near_neighbour function if it can't find any neighbours)

Reply:
We dont need the exception anymore

Reply:
Does that mean we don't need the cutoff line either then?